src/main/webapp/app/course/manage/detail/course-detail-doughnut-chart.component.html

```diff
@@ -1,16 +1,16 @@
 <div class="doughnut mx-3">
     <a *ngIf="titleLink" [routerLink]="['/course-management', course.id, titleLink]">
-        <h5 class="text-center">{{ 'artemisApp.course.detail.' + doughnutChartTitle + 'Title' | artemisTranslate }}</h5>
+        <div class="text-center h5 m-0">{{ 'artemisApp.course.detail.' + doughnutChartTitle + 'Title' | artemisTranslate }}</div>
     </a>
     <h5 *ngIf="!titleLink" class="text-center">{{ 'artemisApp.course.detail.' + doughnutChartTitle + 'Title' | artemisTranslate }}</h5>
     <div class="doughnut-chart-container" [ngClass]="titleLink ? 'clickable' : ''" (click)="openCorrespondingPage()">
         <div class="doughnut-chart-text">
-            <h2 *ngIf="receivedStats" class="text-center">{{ currentPercentage }}%</h2>
-            <h4 *ngIf="receivedStats" class="text-center">{{ currentAbsolute }} / {{ currentMax }}</h4>
-            <h1 *ngIf="!receivedStats" class="text-center" style="z-index: 1">
+            <div *ngIf="receivedStats" class="h4">{{ currentPercentage }}%</div>
```

Comment:
What's the benefit of using a `div` with class `hx`?
Have you considered what would make more sense for accessibility scripts?

Reply:
I do also think that an `hx` solution would be better https://github.com/ls1intum/Artemis/pull/7487#discussion_r1382300224

Reply:
![image](https://github.com/ls1intum/Artemis/assets/78978542/f6bce275-a238-46ec-9a1f-7f4c8f8f326e)
 The headline structure is currently completely broken (on this page?) so I removed the confusing headlines and to fix this structure in another PR.